internal/controller/controllers/local_cluster_import_controller.go

```diff
@@ -35,6 +35,7 @@ const (
 	hubKubeConfigName                                 = "node-kubeconfigs"
 	hubPullSecretNamespace                            = "openshift-config" // #nosec G101
 	hubPullSecretName                                 = "pull-secret"      // #nosec G101
+	importLocalClusterEnabledAnnotation               = "enable-local-cluster-import"
```

Comment:
All of our other [annotation names](https://github.com/openshift/assisted-service/blob/151356e84ce032e003793fa41bf563b5f33d46d5/internal/controller/controllers/agentserviceconfig_controller.go#L89-L93) are prefixed with `agent-install.openshift.io/`. We should do the same here.